src/pages/workspace/expensifyCard/EmptyCardView.tsx

```diff
@@ -6,25 +6,30 @@ import ScrollView from '@components/ScrollView';
 import CardRowSkeleton from '@components/Skeletons/CardRowSkeleton';
 import Text from '@components/Text';
 import useLocalize from '@hooks/useLocalize';
+import useSafeAreaInsets from '@hooks/useSafeAreaInsets';
 import useThemeStyles from '@hooks/useThemeStyles';
 import useWindowDimensions from '@hooks/useWindowDimensions';
+import getPlatform from '@libs/getPlatform';
 import colors from '@styles/theme/colors';
 import CONST from '@src/CONST';
 
 const HEADER_HEIGHT = 80;
 const BUTTON_HEIGHT = 40;
 const BUTTON_MARGIN = 12;
 
+const isIOSNative = getPlatform() === CONST.PLATFORM.IOS;
```

Comment:
Avoid using getPlatform in our App

Reply:
+1 we should refactor into platform specific files